ERROR - 2024-06-23 09:56:19 --> mysqli_sql_exception: Table 'oms.user' doesn't exist in C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `Us...', 0)
#1 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Us...')
#2 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Us...')
#3 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Us...', Array, false)
#4 C:\xampp\htdocs\oms\app\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
#6 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
#7 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\oms\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-06-23 09:56:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'oms.user' doesn't exist
[Method: GET, Route: add-user]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 2 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-23 09:56:19 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'oms.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 4 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-23 09:56:19 --> [Caused by] mysqli_sql_exception: Table 'oms.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 5 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-23 09:56:20 --> mysqli_sql_exception: Table 'oms.user' doesn't exist in C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `Us...', 0)
#1 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Us...')
#2 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Us...')
#3 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Us...', Array, false)
#4 C:\xampp\htdocs\oms\app\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
#6 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
#7 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\oms\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-06-23 09:56:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'oms.user' doesn't exist
[Method: GET, Route: add-user]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 2 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-23 09:56:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'oms.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 4 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-23 09:56:20 --> [Caused by] mysqli_sql_exception: Table 'oms.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 5 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-23 09:57:21 --> mysqli_sql_exception: Table 'oms.user' doesn't exist in C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `Us...', 0)
#1 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Us...')
#2 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Us...')
#3 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Us...', Array, false)
#4 C:\xampp\htdocs\oms\app\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
#6 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
#7 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\oms\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2024-06-23 09:57:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'oms.user' doesn't exist
[Method: GET, Route: add-user]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 2 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\oms\\public\\index.php')
CRITICAL - 2024-06-23 09:57:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'oms.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 4 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\oms\\public\\index.php')
CRITICAL - 2024-06-23 09:57:21 --> [Caused by] mysqli_sql_exception: Table 'oms.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 5 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\oms\\public\\index.php')
ERROR - 2024-06-23 09:57:21 --> mysqli_sql_exception: Table 'oms.user' doesn't exist in C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `Us...', 0)
#1 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Us...')
#2 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Us...')
#3 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Us...', Array, false)
#4 C:\xampp\htdocs\oms\app\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
#6 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
#7 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\oms\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\oms\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2024-06-23 09:57:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'oms.user' doesn't exist
[Method: GET, Route: add-user]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 2 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\oms\\public\\index.php')
CRITICAL - 2024-06-23 09:57:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'oms.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 4 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\oms\\public\\index.php')
CRITICAL - 2024-06-23 09:57:21 --> [Caused by] mysqli_sql_exception: Table 'oms.user' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `User` (`username`, `email`, `password`) VALUES (\'Pratik\', \'pratik@gmail.com\', \'Pratik@123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `User` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 5 APPPATH\Controllers\Sites.php(24): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Sites->insertUser()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sites))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\oms\\public\\index.php')
